### ISSUE TITLE
Create New PIDS 

### ISSUE THREAD

**Issue by klosins** [klosins/SiliconValleyLabor#4]
- [x] Try to fix the overlapping PIDs and create a new variable (Sylvia)

- [ ] Add a dummy var when a SSN number is available, so we know when a PID is reliable (Franklin)

**Comment by zqian1** [comment]
@klosins: Hi Sylvia, Rebecca says she still has to run my updated script to clean raw infutor. Once that's done, I will convert the new data into csv format and pass them to you.

**Comment by klosins** [comment]
@zqian1 sounds good! are we also waiting on that code to get the clean cities?

**Comment by zqian1** [comment]
@klosins: No, I have the code to clean city names modified for the MA 1% sample. I just need to run it today for infutor DI and patent data. Thanks for you patience, I will try to give them to you soon. This week has been extremely hectic so far because I am presenting my work next week.

**Comment by klosins** [comment]
@zqian1 Great! Thanks for letting me know. Good luck on your presentation!

**Comment by zqian1** [comment]
@klosins: 

Can you please take a look at the new datasets i uploaded to dropbox in which I have merged in the cleaned city names for the full DI infutor data and full patent data.

They are at https://www.dropbox.com/preview/SiliconValleyLabor/Data/DI_Infutor/infutor_DI_cityname_cleaned.csv

https://www.dropbox.com/home/SiliconValleyLabor/Data/patent?preview=invpat_full_disambiguation_cityname_cleaned.csv

**Comment by zqian1** [comment]
@klosins: Is this task for creating new pid done btw? I think we can go ahead and close.

**Comment by zqian1** [comment]
@klosins: This task has been superseded by my work of dropping duplicated people for entire infutor data. Closing now.
